careers/ladder.md

```diff
@@ -28,73 +28,183 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
+
+#### Impact
+
+- Learns to complete well-defined and subdivided tasks.
+
+#### Influence
+
 - Develops individual contributor skills, including source control, editors or test-driven development.
+- Seeks assistance and feedback on a regular basis from teammates and manager
+
+#### Discretion
+
+- Receives active guidance from peers and manager.
 
 ### IC1 Associate Engineer
 
+#### tl;dr: focuses on being an effective individual contributor
+
+#### Knowledge Leadership
+
 - Has broad knowledge of CS concepts.
-- Can complete well-defined and subdivided tasks.
 - Focuses on learning tools and processes.
+
+#### Impact
+
+- Can complete well-defined and subdivided tasks.
+
+#### Influence
+
 - Routinely pushes code through the entire lifecycle from development to production.
 - Continues to develop individual contributor skills, including source control, editors or test-driven development.
+- Understands team mission, goals and backlog.
+
+#### Discretion
+
+- Guided by objectives of project or ongoing assignments.
+- Plans own work priorities on a regular basis.
 
 ### IC2 Engineer
 
+#### tl;dr: is an effective team member
+
+#### Knowledge Leadership
+
 - Consistently writes and delivers correct and clean quality code with guidance.
+- Stewards a small-to-medium feature from technical design through completion.
+
+#### Impact
+
+- Makes steady, well-paced progress without the need for constant significant feedback from others.
+
+#### Influence
+
 - Self-sufficient and makes steady progress on tasks.
 - Knows when to ask for help and how to get unblocked.
-- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
-- Owns a small-to-medium feature from technical design through completion.
+- Effectively partners with other engineers and with team's cross-functional partners (including PM, Designer, Data Analyst).
 - Provides help and support outside of area under their responsibility.
 - Participates in the interviewing process of new team members.
 
+#### Discretion
+
+- Actively engages in team rituals.
+- Understands business impact of their work.
+
 ### IC3 Senior Engineer
 
-- Owns a service or large technology component.
-- Requires minimal direction or oversight, delivers quickly.
-- Understands elements of the business that their work supports.
+#### tl;dr: can drive change and impact at the team level
+
+#### Knowledge Leadership
+
 - Understands and makes well-reasoned design decisions and tradeoffs in their area.
 - Works in other areas of the codebase with guidance from owners of that system.
+- Takes responsibility for the maintenance and evolution of a significant service or technology.
+
+#### Impact
+
+- Delivers significant business value at team level.
+- Helps the team deliver on its overall goals.
+- Can drive specific initiatives.
+- Effectively translates ambiguous business problems into manageable tasks.
```

Comment:
I wonder if ‘effectively’ needs some elaboration. Perhaps this should be about creating tasks for _others_ [on the team] to be able to understand effectively?

Reply:
I was thinking of many tickets I see that lack context for others than the author to understand, but perhaps this is a  tangent on more effective communication I general?

Reply:
What about adding "- Empowers team members to take on meaty and interesting tasks." in Influence section?

Reply:
Yeah, I like that 👍